.github/workflows/ci-integration-tests.yml

```diff
@@ -10,7 +10,7 @@ jobs:
   test:
     strategy:
       matrix:
-        python: ['3.7', '3.8', '3.9','3.10', '3.11']
+        python: ['3.7.16', '3.8', '3.9','3.10', '3.11']

```

Comment:
Why do we require specific patch version of python required to be mentioned?

Reply:
In release notes, we need to mention customer should go with 3.7.16 or lower version of Python 3.7 because dbt-teradata is not supporting with 3.7.17 as we have a dependency of_bz2 which 3.7.17 not supporting.

Reply:
If the issue is with 3.7.17 only, we should mention in release documentation that dbt-teradata won't work with 3.7.17 specific.

Reply:
it's an issue with python 3.7.17 only, below is the link:
https://github.com/actions/setup-python/issues/682